Mathematics/MathUtil.cs

```diff
@@ -43,5 +43,44 @@ public static long FindPrimeNumber(int nthPrimeNumberToFind)
 
             return (--a);
         }
+
+        /// <summary>
+        /// Returns the number of digits in an integer.
+        /// Optionally, includes the negative sign in the count.
+        /// </summary>
+        /// <param name="value">The value to evaluate</param>
+        /// <param name="countSign">(default:false) If true, will increase the count by one for all numbers less than 0</param>
+        /// <returns>The number of digits in an integer</returns>
+        /// <remarks>Taken from: https://stackoverflow.com/a/51099524/640196</remarks>
+        public static int GetDigitCount(int value, bool countSign = false)
+        {
+            if (value >= 0)
+            {
+                if (value < 10) return 1;
+                if (value < 100) return 2;
+                if (value < 1_000) return 3;
+                if (value < 10_000) return 4;
+                if (value < 100_000) return 5;
+                if (value < 1_000_000) return 6;
+                if (value < 10_000_000) return 7;
+                if (value < 100_000_000) return 8;
+                if (value < 1_000_000_000) return 9;
+                return 10;
+            }
```

Comment:
Could you just use `Math.Log10`?
```cs
return Math.Floor(Math.Log10(Math.Abs(value))) + 1
```
or is this far more performant or something?

Reply:
Supposedly this is faster...well at least faster the smaller the number is....which seems like the most likely use case for this.

The SO post I linked in the doc comments the guy went through some tests. I haven't taken the time to validate the performance myself.
https://stackoverflow.com/a/51099524/640196